image-search/Models/ActivityIndicator.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ActivityIndicator.swift
+//  image-search
+//
+//  Created by Renato Vieira on 22/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ActivytyIndicator: UIViewController {
+    var activityIndicator = UIActivityIndicatorView()
+        
+    override func loadView() {
+        view                 = UIView()
+        view.backgroundColor = UIColor(white: 0, alpha: 0.8)
+        
+        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
+        activityIndicator.style                                     = UIActivityIndicatorView.Style.whiteLarge
+        activityIndicator.startAnimating()
```

Comment:
You could separate a little bit better this block, adding a space between setting and actions.

Reply:
Done